cpp/include/Ice/Communicator.h

```diff
@@ -93,17 +84,16 @@ namespace Ice
             }
         }
 
-        /// Convert a proxy into a string.
+        /// Converts a proxy into a string.
         /// @param obj The proxy to convert into a stringified proxy.
-        /// @return The stringified proxy, or an empty string if <code>obj</code> is nullopt.
+        /// @return The stringified proxy, or an empty string if `obj` is nullopt.
         /// @see #stringToProxy
         std::string proxyToString(const std::optional<ObjectPrx>& obj) const;
 
-        /// Convert a set of proxy properties into a proxy. The "base" name supplied in the <code>property</code>
-        /// argument refers to a property containing a stringified proxy, such as <code>MyProxy=id:tcp -h localhost -p
-        /// 10000</code>. Additional properties configure local settings for the proxy, such as
-        /// <code>MyProxy.PreferSecure=1</code>. The "Properties" appendix in the Ice manual describes each of the
-        /// supported proxy properties.
+        /// Converts a set of proxy properties into a proxy. The "base" name supplied in the @p property argument refers
```

Comment:
Should this be parameter instead of argument?

```suggestion
        /// Converts a set of proxy properties into a proxy. The "base" name supplied in the @p property parameter refers
```

Reply:
Argument is correct.